docs/make.jl

```diff
@@ -40,7 +40,6 @@ makedocs(sitename = "Catalyst.jl",
     doctest = false,
     clean = true,
     pages = pages,
-    pagesonly = true,
```

Comment:
I know want to build pages that are not actually on the pages list. That also means that I need to move the PEtab doc until we are ready to put that in (when PEtab is updated with respect to the recent changes).